template_content/smart_contracts/{% if deployment_language == 'typescript' %}deploy-config.ts{% endif %}

```diff
@@ -32,10 +32,10 @@ export async function deploy(name: (typeof contracts)[number], appSpec: AppSpec)
     // Edit this to add the custom deployment logic for each contract
     case 'HelloWorldApp':
       const app = await appClient.deploy({
-        allowDelete: isLocal,
-        allowUpdate: isLocal,
-        onSchemaBreak: isLocal ? 'replace' : 'fail',
-        onUpdate: isLocal ? 'update' : 'fail',
+        allowDelete: isMain,
+        allowUpdate: isMain,
```

Comment:
similarly here, I think these two conditions should be inverted yeah?

Reply:
That's correct, allowDelete and allowUpdate should be `!isMain` and likewise for the Python deployment code